client/src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -36,7 +36,7 @@
   max-height: $toolbar-height;
   overflow: hidden;
 
-  @include media-breakpoint-down(sm) {
+  @include media-breakpoint-down(md) {
```

Comment:
https://getbootstrap.com/docs/5.0/migration/#sass

> `media-breakpoint-down()` uses the breakpoint itself instead of the next breakpoint (e.g., `media-breakpoint-down(lg)` instead of `media-breakpoint-down(md)` targets viewports smaller than `lg`).